src/python/pants/engine/internals/graph.py

```diff
@@ -0,0 +1,355 @@
+import os.path
```

Comment:
This file splits out the parts of `legacy/graph.py` that are relevant to V2 code.